addon-test-support/properties/contains.js

```diff
@@ -1,6 +1,5 @@
-import { assign } from '../-private/helpers';
-import { findMany, findOne } from '../extend';
-import { A } from '@ember/array';
+import { assign, every } from '../-private/helpers';
+import { findElementWithAssert } from '../extend';
```

Comment:
internaly we tend to avoid using `findElementWithAssert(` in favour of `findMany` or `findOne`. I'd appreciate if we can avoid `findElementWithAssert` here as well.